packages/framework/src/Model/Localization/LocalizationListener.php

```diff
@@ -33,12 +34,26 @@ class LocalizationListener implements EventSubscriberInterface
      */
     public function __construct(
         Domain $domain,
-        Localization $localization,
-        AdministrationFacade $administrationFacade
```

Comment:
You should use it as a constructor's argument too (as a last one and optional at least). To be able to prepare for next major.

See how it is done for $errorIdProvider
https://github.com/shopsys/shopsys/blob/7fc92f8b1c38c4f28a3607af0d2c7dd6842bbf19/packages/framework/src/Component/Error/NotLogFakeHttpExceptionsExceptionListener.php#L26